models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -55,6 +55,14 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     hostsetup <- paste(hostsetup, sep = "\n", paste(settings$host$prerun, collapse = "\n"))
   }
   
+  # create cdo specific settings
+  cdosetup <- ""
+  if (!is.null(settings$host$cdosetup)) {
+    cdosetup <- paste(cdosetup, sep = "\n", paste(settings$host$cdosetup, collapse = "\n"))
+    jobsh <- gsub("@CDO_SETUP@", cdosetup, jobsh)
```

Comment:
This statement needs to be outside of the if, otherwise this code will leave the `@CDOSETUP@` tag in the job.sh, which will cause the script to fail

Reply:
solved